README.md

```diff
@@ -8,6 +8,20 @@ infrastructure.
 Setup
 -----
 
+### Setting up a GitHub PAT
+
+As a prerequisite for running all scripts in this repo, you will need a GitHub Personal Access
+Token with full access to your account. You can generate one
+[here](https://github.com/settings/tokens).
+
+While creating the token, make sure at the least the scopes `repo`, and `workflow`.
+If you have the rights to manage Biomage repositories, the `admin:org`, `admin:public_key`,
+`admin:repo_hook` and `admin:org_hook` scopes must also be checked for those commands to work. 
+
+Make sure you note down this and supply it when required. 
```

Comment:
I am confused of this sentense, do they have to note the scopes down and when will it be required? Why if they are already on the https://github.com/settings/tokens page, selecting all the scopes they need.